tests/functionaltests/Tests/CortanaTests.mm

```diff
@@ -0,0 +1,103 @@
+//******************************************************************************
+//
+// Copyright (c) 2016 Microsoft Corporation. All rights reserved.
+//
+// This code is licensed under the MIT License (MIT).
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+// THE SOFTWARE.
+//
+//******************************************************************************
+
+#include <TestFramework.h>
+#include <Foundation/Foundation.h>
+#include <COMIncludes.h>
+#include "MockClass.h"
+#include <windows.applicationModel.activation.h>
+#include <windows.media.speechRecognition.h>
+#include <COMIncludes_End.h>
+#include <UWP/WindowsApplicationModelActivation.h>
+#include <UWP/WindowsMediaSpeechRecognition.h>
+#include <UWP/WindowsUIXaml.h>
+#include "StringHelpers.h"
+#include "UIKit/UIApplication.h"
+
+using namespace ABI::Windows::ApplicationModel::Activation;
+using namespace ABI::Windows::Media::SpeechRecognition;
+using namespace Microsoft::WRL;
+
+MOCK_CLASS(MockSpeechRecognitionResult,
+           public RuntimeClass<RuntimeClassFlags<WinRtClassicComMix>, ISpeechRecognitionResult, ISpeechRecognitionResult2> {
+               InspectableClass(RuntimeClass_Windows_Media_SpeechRecognition_SpeechRecognitionResult,
+                                BaseTrust); // claim to be the implementation for the real system RuntimeClass for SpeechRecognitionResult.
```

Comment:
Nit: I would prefer these comment lines to go on a separate line before this. The line itself is long enough as is. Additionally coding conventions state that the first word of the comment should be capitalized as a comment is lexically an english sentence.